packages/cfd/src/Components/props.types.ts

```diff
@@ -68,6 +68,68 @@ export type TTradingPlatformAvailableAccount = {
     sub_account_type: string;
 };
 
+export type TCardFlipStatus = {
+    svg: boolean;
+    bvi: boolean;
+    labuan: boolean;
+    vanuatu: boolean;
+    maltainvest: boolean;
+};
+
+export type TClickableDescription = {
+    type: 'text' | 'link';
+    text: string;
+};
+
+export type TJurisdictionCardSectionTitleIndicators = {
+    type: 'displayText' | 'displayIcons';
+    display_text?: string;
+    display_text_skin_color?: string;
+    display_icons?: Array<string>;
+};
+
+export type TJurisdictionCardSection = {
+    key: string;
+    title: string;
+    title_indicators?: TJurisdictionCardSectionTitleIndicators;
+    description?: string;
+    clickable_description?: Array<TClickableDescription>;
+};
+
+export type TJurisdictionCardItemVerification = Array<
+    'document_number' | 'selfie' | 'identity_document' | 'name_and_address'
+>;
+
+export type TJurisdictionCardItems = {
+    header: string;
+    over_header?: string;
+    synthetic_contents: Array<TJurisdictionCardSection>;
+    financial_contents: Array<TJurisdictionCardSection>;
+    is_over_header_available: boolean;
+    synthetic_verification_docs?: TJurisdictionCardItemVerification;
+    financial_verification_docs?: TJurisdictionCardItemVerification;
+};
+
+export type TJurisdictionCardBackRequiredDocsItem = {
+    icon: string;
+    text: string;
+};
+
+export type TJurisdictionCardBackSectionRequiredDocs = {
+    document_number?: TJurisdictionCardBackRequiredDocsItem;
+    selfie?: TJurisdictionCardBackRequiredDocsItem;
+    identity_document?: TJurisdictionCardBackRequiredDocsItem;
+    name_and_address?: TJurisdictionCardBackRequiredDocsItem;
+};
+
+type TJurisdictionVerificationColors = 'yellow' | 'red' | 'green';
+
+export type TJurisdictionCardBackSectionStatusReferences = {
```

Comment:
why do we need both 
TJurisdictionCardBackSectionStatusReferences and TJurisdictionCardBackSectionRequiredDocs?

Reply:
@amina-deriv both are constant values, but one is an array in which all elements will be displayed and the other is an object from which only required items will be displayed. do you have any refactoring ways in mind?
<img width="639" alt="Screenshot 2023-04-07 at 3 45 25 PM" src="https://user-images.githubusercontent.com/122449658/230603611-d141b884-c274-4040-a9d7-090deef13244.png">